### PR TITLE
formatting the updated date to be iso format

### PR DESCRIPTION
**Pull request by goldblatt**
This PR addresses issue https://github.com/covid-projections/covid-data-model/issues/206

### Please Check
- [ ] Will the *current* schema in [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.md) be updated with this PR?
- [ ] Are tests passing?

### CODE REVIEW

File: cli/run_top_counties_dataset.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 
 @click.command("deploy-top-counties")
 @click.option(
-    "--run_validation",
-    "-r",
-    default=True,
+    "--disable_validation",
```
Comment (ghop02) [inline cli/run_top_counties_dataset.py:13]: most of the others are `--disable-validation` (click converts the name to th underscore in the function calls)

File: libs/functions/generate_api.py [hunk 2]
```diff
@@ -7,14 +8,24 @@
 from libs.datasets import results_schema as rc
 
 
+def _format_date(input_date):
+    if not input_date: 
+        raise Exception("Can't format a date that doesn't exist")
+    if isinstance(input_date, str): 
+        # note if this is already in iso format it will be grumpy. maybe use dateutil
+        datetime_obj = datetime.strptime(input_date, '%m/%d/%Y %H:%M')
```
Comment (ghop02) [inline libs/functions/generate_api.py:16]: Where does this format type come from? would be great to make everything iso earlier in the flow.

there's also this new function that might be helpful: https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat

File: cli/run_top_counties_dataset.py (hunk 1 above)
Comment (goldblatt) [reply cli/run_top_counties_dataset.py:13]: coool done

File: libs/functions/generate_api.py (hunk 2 above)
Comment (goldblatt) [reply libs/functions/generate_api.py:16]: ya it's in the JHU dataset we can format to ISO there although I think that PR is in flight @ghop02 wanna fix it in your PR?

Comment (ghop02) [reply libs/functions/generate_api.py:16]: Yeah sounds good!

**Pull request merged**